features/visual-editor.feature

```diff
@@ -10,15 +10,49 @@ Feature: Save edition content with visual editor
     Then I should see the visual editor instead of the govspeak editor
     When I fill in the required fields for publication "Publication with visual editor" in organisation "Visual ministry"
     And I save and go to document summary
-    Then I see the visual editor on subsequent edits of the publication
+    Then I should see the visual editor on subsequent edits of the publication
     And I force publish the publication "Publication with visual editor"
-    
+
+  @javascript
+  Scenario: I create a new publication and exit the visual editor experience
+    When I start creating a new publication
+    Then I should see the visual editor instead of the govspeak editor
+    When I fill in the required fields for publication "Publication with visual editor" in organisation "Visual ministry"
+    And I exit the visual editor experience
+    Then I should see the govspeak editor instead of the visual editor
+    When I save and go to document summary
+    Then I should see the govspeak editor on subsequent edits of the publication
+
+  @javascript
+  Scenario: I edit a publication saved with visual editor
+    When I edit a publication saved with visual editor
+    Then I should see the visual editor instead of the govspeak editor
+    When I update the publication in the visual editor
+    And I exit the visual editor experience
+    And I save and go to document summary
+    Then I should see the govspeak editor on subsequent edits of the publication
+
+  @javascript
+  Scenario: I edit a publication that has been previously exited
+    When I edit a publication that has been previously exited
+    Then I should see the govspeak editor
+    When I update the publication in the govspeak editor
+    And I save and go to document summary
+    Then I should see the govspeak editor on subsequent edits of the publication
```

Comment:
I feel these lines are a bit redundant, but I can understand if you want to keep them around for completeness sake. Likewise for the test below about pre-existing documents

Reply:
I think it is good to call out that a user with permission does not have the ability to revert the flag set on the document (be it pre-existing or exited).